collector.go

```diff
@@ -21,6 +21,10 @@ var (
 		"nextcloud_files_total",
 		"Number of files served by the instance.",
 		nil, nil)
+	freeSpaceDesc = prometheus.NewDesc(
+		"nextcloud_freespace",
```

Comment:
I think I'd add the unit of the metric to its name: `nextcloud_free_space_bytes`

Reply:
You're right, I changed it.